includes/embeds/class-amp-youtube-embed.php

```diff
@@ -64,6 +64,12 @@ public function shortcode( $attr ) {
 	}
 
 	public function oembed( $matches, $attr, $url, $rawattr ) {
+		// Use the standard oembed handler if it's a playlist
+		$parsed_url = wp_parse_url( $url );
+		if ( isset( $parsed_url['path'] ) && false !== strpos( $parsed_url['path'], 'playlist' ) ) {
+			return wp_oembed_get( $url );
```

Comment:
Oh no, you're right -- I got mixed up :)

Is there a way we can just bail out of our custom oembed handler and fallback directly to core's oembed handler instead?